internal/Get-Connection.ps1

```diff
@@ -0,0 +1,115 @@
+﻿Function Get-Connection
```

Comment:
I really like this command and how many failbacks it has. I'm happy to remove Resolve-IpAddress but Get-Connection isn't right. This does a lot of resolutions -- I think it should be Resolve-NetworkInfo (Similar to Microsoft's Resolve-DnsName)

Also, what do you think about getting the Kerberos/SPN name/entry too? I'd need that in another command.